### ISSUE TITLE
Add a default ToString behaviour on CheckLinks

### ISSUE THREAD

**Issue by rhwy** [tpierrain/NFluent#230]
It should be nice that a CheckLink returns a value like "ok" or "success" when a ToString is called on it instead of having its default class naming. 

For exemple when you use NFluent as a de facto test framework in Xamarin workbooks, the result of the last instruction of a block is printed. It's perfect when it fails:

<img width="624" alt="banners_and_alerts_et_tdd-with-nfluent-in-a-book" src="https://user-images.githubusercontent.com/212965/36774987-6f878ea0-1c60-11e8-97af-2a106b641c24.png">

But it would be better to print "success" instead of class name when it's ok:

<img width="548" alt="tdd-with-nfluent-in-a-book" src="https://user-images.githubusercontent.com/212965/36774897-2987ece2-1c60-11e8-90da-06d4b4bd4d5e.png">

**Comment by dupdob** [comment]
Dev done, beta to be released

**Comment by rhwy** [comment]
👍🙃😘

**Comment by dupdob** [comment]
Just published a beta  (V2.3.0-156) with this behavior.
You can try it and give feedback, please?
```csharp
Install-Package NFluent -Version 2.3.0-beta-0156 -Source https://www.myget.org/F/dupdobnightly/api/v3/index.json
